.circleci/config.yml

```diff
@@ -122,8 +128,9 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-            - v3-dependencies-{{ checksum "Pipfile.lock" }}
-      - run: *install-pipenv
+            - v4-dependencies-{{ checksum "uv.lock" }}
+      - install_uv
+      - run: uv venv
```

Comment:
..and another one here

Reply:
see https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/117#discussion_r1890083938